x-pack/plugins/security_solution/public/detections/components/rules/step_about_rule/index.test.tsx

```diff
@@ -223,32 +224,33 @@ describe('StepAboutRuleComponent', () => {
       .first()
       .simulate('change', { target: { value: '80' } });
 
-    wrapper.find('[data-test-subj="about-continue"]').first().simulate('click').update();
-    await waitFor(() => {
-      const expected: Omit<AboutStepRule, 'isNew'> = {
-        author: [],
-        isAssociatedToEndpointList: false,
-        isBuildingBlock: false,
-        license: '',
-        ruleNameOverride: '',
-        timestampOverride: '',
-        description: 'Test description text',
-        falsePositives: [''],
-        name: 'Test name text',
-        note: '',
-        references: [''],
-        riskScore: { value: 80, mapping: [], isMappingChecked: false },
-        severity: { value: 'low', mapping: fillEmptySeverityMappings([]), isMappingChecked: false },
-        tags: [],
-        threat: [
-          {
-            framework: 'MITRE ATT&CK',
-            tactic: { id: 'none', name: 'none', reference: 'none' },
-            technique: [],
-          },
-        ],
-      };
-      expect(stepDataMock.mock.calls[1][1]).toEqual(expected);
+    await act(async () => {
```

Comment:
Note: by wrapping the button click event inside the `await act`, we don't need the `waitFor` call below. This allows us to get a proper error from the test instead of a jest timeout.

<img width="395" alt="Screenshot 2020-08-25 at 10 36 52" src="https://user-images.githubusercontent.com/2854616/91190566-034ce080-e6f4-11ea-8843-4957b3a4b72f.png">

Reply:
That should be the only use of `waitFor` in this file.

Reply:
Yes it seems so but I didn't touch the other tests that also wrap the expect inside a `waitFor`. This test was the one failing and the timeout error did not let me see _why_ it was failing.

Reply:
Oh, absolutely! I just meant that the `import` is now unused.